conf/messages

```diff
@@ -960,6 +971,7 @@ label.status=Status
 label.search=Search
 label.clear.search=Clear Search
 
+label.Address=<h1 class="govuk-heading-l">Address<h1/>
```

Comment:
why do we have html tags in here? They should be in the twirl template

Reply:
@darrell-plant its used like this:  <th class="govuk-table__header">@messages("enrolment.organisation.name.label")</th> 

so I thought i'll put it in the message. 

Please let me know if this is not correct

Reply:
not correct :)

Reply:
also, follow existing conventions
`label.Address` should be `label.address`